GameServerLib/Game.cs

```diff
@@ -110,14 +117,54 @@ public void Initialize(ushort port, string blowfishKey, Config config)
             _pauseTimer.Elapsed += (sender, args) => PauseTimeLeft--;
             PauseTimeLeft = 30 * 60; // 30 minutes
 
+            // TODO: GameApp should send the Response/Request handlers
             _packetServer = new PacketServer();
-            _packetServer.InitServer(port, blowfishKey, this);
+            _packetServer.InitServer(port, blowfishKey, this, RequestHandler, ResponseHandler);
```

Comment:
Maybe pass ``_packetServer`` itself as argument to ``Game.Initialize`` as to remove the dependancy of ``Game`` on ``address``, ``blowfish`` and ``port``?

Reply:
Changed.